manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml

```
@@ -1,23 +1,24 @@
-# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.7-3-4.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
-
-PackageIdentifier: Glodon.CADReader.CN
-PackageVersion: 5.19.0.91
-InstallerType: inno
-Scope: machine
-UpgradeBehavior: install
-FileExtensions:
-- bin
-- dwg
-- dxf
-- octet-stream
-AppsAndFeaturesEntries:
-- DisplayName: CAD快速看图v5.19.0.91
-  ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
-Installers:
-- Architecture: x86
-  InstallerUrl: http://cdn.cad.glodon.com/app/CADReaderPc_5.19.0.91.exe
-  InstallerSha256: E0BDE88721BC81A9F58D36D5174DFF7B80AF4F7805C98FEBA0D80F6FC3792274
-  ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
-ManifestType: installer
-ManifestVersion: 1.4.0
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Glodon.CADReader.CN
+PackageVersion: 5.19.0.91
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- bin
+- dwg
+- dxf
+- octet-stream
+ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
+ReleaseDate: 2023-04-20
+AppsAndFeaturesEntries:
+- DisplayName: CAD快速看图v5.19.0.91
+  ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
+Installers:
+- Architecture: x86
+  InstallerUrl: http://yuntu-f.kuaicad.com/app/CADReaderPc_5.19.0.91.exe

```

```suggestion
  InstallerUrl: https://yuntu-f.glodon.com/app/CADReaderPc.exe
```
This is the download URL from the PackageUrl page.